### PR TITLE
exclude shrinkwrap from openapi and bump vert.x in build-parent

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

**Review by cescoffier — APPROVED** [review]
LGTM, need to check why the CI has failed.

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -517,6 +517,12 @@
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-smallrye-openapi</artifactId>
                 <version>${project.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.jboss.shrinkwrap</groupId>
+                        <artifactId>shrinkwrap-api</artifactId>
+                    </exclusion>
+                </exclusions>
```
Comment (gsmet) [inline bom/runtime/pom.xml]: Wondering if it's an issue in `smallrye-openapi` itself and if the dependency should be test there?

Comment (gastaldi) [reply bom/runtime/pom.xml]: ShrinkWrap is used by this class in SmallRye OpenAPI: https://github.com/smallrye/smallrye-open-api/blob/master/implementation/src/main/java/io/smallrye/openapi/api/util/ArchiveUtil.

Comment (gsmet) [reply bom/runtime/pom.xml]: I wonder if this should be a test class or if it's really supposed to be used in user applications.

@EricWittmann ?

**Comment by gsmet** [comment]
I merged but let's try to get to the bottom of it on the SmallRye OpenAPI side.

**Comment by gastaldi** [comment]
I created https://github.com/smallrye/smallrye-open-api/pull/249 moving that class to the TCK project (which seems to be the only place where it is used)

**Comment by gsmet** [comment]
Thanks! That's exactly what I suspected.

**Comment by EricWittmann** [comment]
Well, we're having this discussion here:  https://github.com/smallrye/smallrye-open-api/issues/246

It turns out that Thorntail uses the `ArchiveUtil` class, so it can't be removed.  The latest proposal is to mark ShrinkWrap as optional...which means consumers of SmallRye OpenAPI will need to BYO-ShrinkWrap. :)

File: bom/runtime/pom.xml (hunk 1 above)
Comment (EricWittmann) [reply bom/runtime/pom.xml]: See my comment in this PR.

**Pull request merged**